common/reducers/Filter.js

```diff
@@ -11,7 +11,8 @@ module.exports = function Filter(prevState = INITIAL_STATE, action) {
   switch (action.type) {
```

Comment:
It seems like we could do even better here by only creating the state object inside the NOTIFY_FILTER_QUERY, so that it never gets created at all in the default cause, since it will be unused...